packages/db/src/async_operations.ts

```diff
@@ -0,0 +1,19 @@
+import { toSnake } from 'snake-camel';
+import { ApiAsyncOperation } from '@cumulus/types/api/async_operations';
+import { PostgresAsyncOperation } from './types';
+
+export const translateApiAsyncOperationToPostgresAsyncOperation = (
+  record: ApiAsyncOperation
+): PostgresAsyncOperation => {
+  const translatedRecord: PostgresAsyncOperation = <PostgresAsyncOperation>toSnake(record);
+  if (record.output !== undefined) {
+    translatedRecord.output = record.output;
+  }
+  if (record.createdAt !== undefined) {
+    translatedRecord.created_at = new Date(record.createdAt);
+  }
+  if (record.updatedAt !== undefined) {
```

Comment:
I think we can pull out these two `if` statements into a helper function, but I don't know where it would live.

Reply:
I think this is fine if we create <a file> (e.g. 'translation-utils.ts') or some better-er name that is explicitly not user facing within this package for abstraction purposes. 

Alternately we could go with a micro-package and make lerna hate us, or use @common (yuck). 

I have a separate question about why this is needed, but I'll ask when I do the review.

Reply:
Why: `number` vs `Date` types on the timestamps between the `Api*` and `Postgres*` types.

Reply:
This kinda feels like a micro-optimization at this point. I would just leave it and if we're repeating this logic all over the place, we can consolidate

Reply:
I think I can agree with that 👍